include/fastdds/dds/subscriber/Subscriber.hpp

```diff
@@ -294,11 +322,15 @@ class Subscriber : public DomainEntity
             const std::string& profile_name,
             DataReaderQos& qos) const;
 
-    /* TODO
-       bool copy_from_topic_qos(
-            DataReaderQos& reader_qos,
-            const fastrtps::TopicAttributes& topic_qos) const;
+    /**
+     * @brief Copies TopicQos into the corresponding DataReaderQos
+     * @param[out] reader_qos
+     * @param[in] topic_qos
+     * @return RETCODE_OK if successful, an error code otherwise
      */
+    RTPS_DllAPI ReturnCode_t copy_from_topic_qos(
```

Comment:
This method could be `static`, as it does not need any Subscriber instance. If we want to keep it as non-static, at least make it `const`